packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -224,7 +252,9 @@ const EpicBody: React.FC<BodyProps> = ({
     );
 };
 
-const ContributionsEpic: React.FC<EpicProps> = ({
+export const getContributionsEpic: (
```

Comment:
`getContributionsEpic` is used to build 2 different epic modules. This is because variant 2 requires a different UI